utils/tzif/src/lib.rs

```diff
@@ -30,7 +30,6 @@ use data::{posix::PosixTzString, tzif::TzifData};
 use error::Error;
 use std::fs::File;
 use std::path::Path;
-
```

Comment:
revert?

Reply:
Done. Not sure why this diff isn't showing as outdated because this change doesn't exist after I synced.